kyo-data/shared/src/main/scala/kyo/Maybe.scala

```diff
@@ -445,5 +439,7 @@ object Maybe:
                 else
                     new PresentAbsent(depth)
         end PresentAbsent
+
+        def constAbsent(ignored: Any): Absent = Absent
```

Comment:
We can also avoid the function allocation to invoke `applyOrElse`:
```suggestion
        private val constAbsent: Any => Absent = _ => Absent
```